inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -4,11 +4,19 @@
    SPDX-License-Identifier: Apache-2.0
 """
 import argparse
+import logging
 
+from ..inbc_exception import InbcException
 from ..xml_tag import create_xml_tag
 
+logger = logging.getLogger(__name__)
```

Comment:
Add logging to indicate the successful addition or removal of a source, which can be helpful for debugging and auditing purposes. [medium]